interfaces/builtin/serial_port.go

```diff
@@ -77,7 +77,7 @@ func (iface *serialPortInterface) String() string {
 //   - ttyHSX (Qualcomm GENI based QTI serial cores)
 //   - ttyGSX (USB gadget serial devices)
 //   - ttyLPX (NXP Layerscape SoC UART serial ports)
-var serialDeviceNodePattern = regexp.MustCompile("^/dev/tty(mxc|USB|ACM|AMA|XRUSB|S|O|SC|MSM|HS|GS|LP)[0-9]+$")
+var serialDeviceNodePattern = regexp.MustCompile("^/dev/tty(mxc|USB|ACM|AMA|XRUSB|S|O|SC|MSM|HS|GS|LP|RPMSG)[0-9]+$")
```

Comment:
Please add comment about device `- ttyRPMSG  (... devices)` as for others

Reply:
thanks, done.